game_board/database/game_board_db.py

```diff
@@ -112,3 +112,27 @@ def list_games():
         cursor: to iterate game ids
     """
     return client.InitialDB.Active_Games.find({},{'_id':0, 'game_id': 1})
+
+def purge_old_games():
+    """
+    Checks the datetime of games, deletes anything older than 1 day
+
+    Parameters:
+    None
+
+    Returns:
+        int: 0
+    """
+    yesterday = (dt.datetime.now() - dt.timedelta(days=1))
+
+    mongo_cursor = client.InitialDB.Active_Games.find({},{'_id':0, 'game_id': 1, 'time_created':1})
+
+    for game in mongo_cursor:
+        try:
+            game_date = game['time_created']
+            game_datetime_obj = dt.datetime.strptime(game_date,"%d/%m/%Y %H:%M:%S")
+            if game_datetime_obj < yesterday:
```

Comment:
Will this delete my game if I start it at 11:59 and if this code runs after midnight 00:00?

Reply:
This calculates the current time of running the function and then subtracts 24 hours ( this is what dt.timedelta(days=1) is ) , so it doesn't really matter which minute it falls on of a day. Prior to 24 hours will be deleted, but not anything sooner.